.travis.yml

```diff
@@ -2,20 +2,22 @@ env:
  # Note: We have to use Cabal 2.4 with GHC 7.0 since for some reason
  # builds of "text" fail with GHC 7.0 and Cabal versions prior to 2.4
  # https://github.com/haskell/bytestring/issues/213
```

Comment:
Nit: Maybe remove this and add a reference to the issue in the link below

Reply:
Not sure what you want here, can suggest a diff?

Reply:
Apologies, please mark resolved. Not sure where I was going there anymore. :/